FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+TableView.swift

```diff
@@ -77,7 +77,7 @@ extension ComposeViewController: ASTableDelegate, ASTableDataSource {
             case let (.searchEmails(recipients), .searchResults):
                 guard indexPath.row > 0 else { return DividerCellNode() }
                 guard recipients.isNotEmpty else { return self.noSearchResultsNode() }
-                guard let recipient = recipients[safe: indexPath.row-1] else { return ASCellNode() }
+                guard let recipient = recipients[safe: indexPath.row - 1] else { return ASCellNode() }
```

Comment:
It seems this line contains just code formatting. Does your IDE support auto formating before committing?

Reply:
Yes, we use `swiftlint`, but in this PR I updated it's config with additional rules like this one - spaces around operators.